.github/workflows/ci-build.yaml

```
@@ -137,7 +137,7 @@ jobs:
           go-version: "1.21"
           cache: true
       # windows run does not use makefile target because it does a lot more than just testing and is not cross-platform compatible
-      - run: go test -p 1  -covermode=atomic -coverprofile='coverage.out' $(go list ./... | select-string -Pattern 'github.com/argoproj/argo-workflows/v3/workflow/controller' , 'github.com/argoproj/argo-workflows/v3/server' -NotMatch)
+      - run: go test -covermode=atomic -coverprofile='coverage.out' $(go list ./... | select-string -Pattern 'github.com/argoproj/argo-workflows/v3/workflow/controller' , 'github.com/argoproj/argo-workflows/v3/server' -NotMatch)
```

```suggestion
      - run: go test -p 20 -covermode=atomic -coverprofile='coverage.out' $(go list ./... | select-string -Pattern 'github.com/argoproj/argo-workflows/v3/workflow/controller' , 'github.com/argoproj/argo-workflows/v3/server' -NotMatch)
```

The Linux ones use `20`, so I would think we should match them, no?

---

Also, can't comment on unchanged lines, but should we decrease `timeout-minutes` now? May want to test more to get an average though as GH runs are not all equivalent

---

Let's leave it as is and observe.